.tekton/windows-machine-config-operator-bundle-master-pull-request.yaml

```diff
@@ -35,6 +35,15 @@ spec:
     value: quay.io/redhat-user-workloads/windows-machine-conf-tenant/windows-machine-config-operator/windows-machine-config-operator-bundle-master:on-pr-{{revision}}
   - name: revision
     value: '{{revision}}'
+  taskRunSpecs:
+    - pipelineTaskName: build-container
```

Comment:
Looks like this is bumping build-container.
Aren't we OOMing on `build-source-image`?

Reply:
Do we need to increase both?

Reply:
I think you're right, that the source image should be the one that's getting bumped. Let's see how the test runs :+1: